sys/src/lib.rs

```diff
@@ -9,6 +9,17 @@ pub mod platform;
 #[cfg(feature = "mint")]
 mod mint_impls;
 
+/// # Safety
+///
+/// No this is actually safe but we need the type to match.
+pub unsafe extern "system" fn stub_enumerate_api_layer_properties(
```

Comment:
There's no reason for this to be public, or defined in a different crate than it's used in.

Reply:
this is used in `openxr` though, do you want me to duplicate it in both places? or do you want me to add a `#[doc(hidden)]`

Reply:
oh right, got a bit confused by where the generator code was going.